docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -410,6 +410,54 @@
                     </td>
                 </tr>
                 <tr>
+                                              <!-- change link to StandAlone design/tutorial doc -->
+                <th colspan="3" class="section" id="StandAlone"><a href="../index.html">StandAlone job configuration</a></th>
+                </tr>
+                <tr>
+                    <td class="property" id="coordinator.zk.connect">coordinator.zk.connect</td>
+                    <td class="default"></td>
+                    <td class="description">
+                        <strong>Required </strong> for Zookeeper-based StandAlone applicaiton. Zookeeper coordinates to be used by
+                        the application coordination. Usually in "host:port" format.
+                    </td>
+                </tr>
+                <tr>
+                    <td class="property" id="coordinator.zk.session-timeout-ms">coordinator.zk.session-timeout-ms</td>
+                    <td class="default"> 30000 </td>
+                    <td class="description">
+                        Zookeeper session timeout for all the ZK connections in milliseconds.
+                    </td>
+                </tr>
+                <tr>
+                    <td class="property" id="coordinator.zk.connection-timeout-ms">coordinator.zk.connection-timeout-ms</td>
+                    <td class="default"> 60000 </td>
+                    <td class="description">
+                        Zookeeper connection timeout in milliseconds.
+                    </td>
+                </tr>
+                <tr>
+                    <td class="property" id="coordinator.zk.new-jobmodel-consensus-timeout-ms">coordinator.zk.new-jobmodel-consensus-timeout-ms</td>
+                    <td class="default"> 40000 </td>
+                    <td class="description">
+			How long the processors will wait until all of them report acceptance of a the new job model before rolling back.
+                    </td>
+                </tr>
+                <tr>
+                    <td class="property" id="job.coordinator.factory">job.coordinator.factory</td>
+                    <td class="default"></td>
+                    <td class="description">
+			Class to use for job coordination. Currently available values are:
+                       <dl>
+                            <dt><code>org.apache.samza.standalone.StandaloneJobCoordinatorFactory</code></dt>
+                            <dd>For use in StandAlone, no Zookeeper, fixed partition mapping application. </dd>
```

Comment:
This is the reason I suggested not to use "StandAlone" above. You are saying "StandAlone, no Zookeeper" and previously, it had zk.connect config etc for Zookeeper based coordination. So, let's just try not use "standalone" as much as we can.